docsite/rst/playbooks_strategies.rst

```diff
@@ -0,0 +1,39 @@
+Strategies
+===========
+
+In 2.0 we added a new way to control play execution, ``strategy``, by default plays will
+still run as they used to, with what we call the ``linear`` strategy. All hosts will run each
+task befor any host starts the next task, using the number of forks (default 5) to parallelize.
+
+The ``serial`` directive can 'batch' this behaviour to a subset of the hosts, which then run to
+completion of the play before the next 'batch' starts.
+
+A second ``strategy`` ships with ansible ``free``, which allows each host to run until the end of
+the play as fast as it can.::
+
```

Comment:
This section seems like it is sufficient for people who are already familiar with ansible-1.x's parallelism but needs some thing for people who are new.  It seems like it would be easy for people to miss that linear is parallel within a task and that serial is related to that within-a-play parallelism.

free's parallelism could then be defined in contrast to that.  I'm also not sure what effect serial might have if the strategy was free.  Is that an error?  Does it force linear strategy behaviour?